test/client/clients/WebsocketClient.test.js

```diff
@@ -60,3 +66,99 @@ describe('WebsocketClient', () => {
     });
   });
 });
+
+describe('websocket client injection', () => {
```

Comment:
This should be moved to the `test/server` directory. Either in `Server.test.js`, or possibly a new test file of its own. @evilebottnawi thoughts on where it should go exactly?

Reply:
@EslamHiko yep, let's move this test to `test/server`
@Loonride maybe `test/server/hot-and-liveReload-integration`, no idea how better naming this :smile:

Reply:
@Loonride @evilebottnawi okay, will move it to `test/server/socket-injection` what do you think about the name? 😅

Reply:
`test/server/socket-injection.test.js` :+1: